shared/__mocks__/feature-flags.js

```diff
@@ -7,6 +7,7 @@ if (!__STORYBOOK__) {
 }
 
 const ff: FeatureFlags = {
+  addIdentity: true,
```

Comment:
this feature flag should be used for all the generic proofs so it should likely be about that, not just this one button

Reply:
:white_check_mark: